### ISSUE TITLE
Comics can be downloaded but not read if "Allow management of all files" is not granted

### ISSUE THREAD

**Issue by SebiderSushi** [tachiyomiorg/tachiyomi#9546]
### Steps to reproduce

1. Install Tachiyomi
2. When asked to allow acces to all files, click deny
3. Go to Settings > Downloads > Download location > Custom location
4. Choose a Directory in the DocumentsProvider dialog and grant SAF access
5. Add a comic
6. Download a chapter
7. Once the download is finished, open the downloaded chapter

### Expected behavior

If Tachiyomi is able to write to the download directory via SAF it should also be able to read from it, even without the "Allow management of all files" permission.

As a user, i expect an application to require either an All Files permission or SAF access, not both (unless explicitly required for some quirky use-case or faulty OS implementation)

The [FAQ mentions some performance caveats](https://tachiyomi.org/help/faq/#what-does-it-mean-for-tachiyomi) when using SAF. As a user, i expect to be able to choose whether the performance hit is acceptable on my device.

### Actual behavior

The `.cbz` file for the chapter appears in the download directory as expected. When opened with [MuPDF](https://f-droid.org/en/packages/com.artifex.mupdf.viewer.app/) each page is shown without issue.

However, in Tachiyomi, the reader does not open. Instead, Tachiyomi displays a toast containing `[file path to the chapter] (permission denied)`  
The path shown in the toast is the native path starting with `/storage/emulated/0/`, not a content URI.

Once the "Allow management of all files" permission is granted, Tachiyomi behaves as expected.

### Tachiyomi version

Stable 0.14.6 (official build)

### Android version

LineageOS 18.1 (Android 11)

### Device

OnePlus X

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
